data-prepper-plugins/aws-plugin-api/src/main/java/org/opensearch/dataprepper/aws/api/AwsCredentialsOptions.java

```diff
@@ -122,6 +136,17 @@ public Builder withStsHeaderOverrides(final Map<String, String> stsHeaderOverrid
             return this;
         }
 
+        /**
+         * Configures whether to use default credentials.
+         *
+         * @param useDefaultCredentials
+         * @return The {@link Builder} for continuing to build
+         */
+        public Builder withUseDefaultCredentials(final boolean useDefaultCredentials) {
```

Comment:
```suggestion
        public Builder withUseDefaultCredentialsProvider(final boolean useDefaultCredentialsProvider) {
```

Reply:
Thanks @dlvenable. I have made the name change.